examples/E2EBert/README.md

```diff
@@ -87,21 +88,22 @@ Create an empty directory `model_store` and run the following command to start T
 
 Run the following command to create a new deployment named `news_classification_test`
 
-`mlflow deployments create -t torchserve -m state_dict.pth --name news_classification_test -C "MODEL_FILE=news_classifier.py" -C "HANDLER=news_classifier_handler.py" -C "EXTRA_FILES=class_mapping.json,bert_base_uncased_vocab.txt"`
+`mlflow deployments create -t torchserve -m state_dict.pth --name news_classification_test -C "MODEL_FILE=news_classifier.py" -C "HANDLER=news_classifier_handler.py" -C "EXTRA_FILES=class_mapping.json,bert_base_uncased_vocab.txt,wrapper.py"`
 
 Note: TorchServe plugin determines the version number by itself based on the deployment name. hence, version number 
 is not a mandatory argument for the plugin. For example, the above command will create a deployment `news_classification_test` with version 1.
 
 If needed, version number can also be explicitly mentioned as a config variable.
 
-`mlflow deployments create -t torchserve -m state_dict.pth --name news_classification_test -C "VERSION=1.0" -C "MODEL_FILE=news_classifier.py" -C "HANDLER=news_classifier_handler.py" -C "EXTRA_FILES=class_mapping.json,bert_base_uncased_vocab.txt"`
+`mlflow deployments create -t torchserve -m state_dict.pth --name news_classification_test -C "VERSION=1.0" -C "MODEL_FILE=news_classifier.py" -C "HANDLER=news_classifier_handler.py" -C "EXTRA_FILES=class_mapping.json,bert_base_uncased_vocab.txt,wrapper.py"`
 
 
-## Running prediction based on deployed model
+## Running prediction and explain based on deployed model
 
 For testing the fine tuned model, a sample input text is placed in `input.json`
 Run the following command to invoke prediction of our sample input 
 
 `mlflow deployments predict --name news_classification_test --target torchserve --input-path input.json  --output-path output.json`
+`mlflow deployments explain --name news_classification_test --target torchserve --input-path input.json  --output-path output.json`
```

Comment:
@Arvind-Ideas2IT Please check the formats in the md what shows up, there no space between these two command.

Reply:
@HamidShojanazeri Done.